packages/neuron/package.json

```diff
@@ -10,6 +10,7 @@
     "update:locales": "i18next"
   },
   "dependencies": {
+    "@ckb-lumos/base": "0.23.0",
```

Comment:
`@ckb-lumos/lumos` can be imported to avoid importing various sub-packages in the future.